pkg/whiteflag/white_flag.go

```diff
@@ -231,11 +257,55 @@ func ComputeWhiteFlagMutations(s *storage.Storage, msIndex milestone.Index, cach
 		for _, input := range inputOutputs {
 			delete(wfConf.NewOutputs, string(input.OutputID()[:]))
 			wfConf.NewSpents[string(input.OutputID()[:])] = utxo.NewSpent(input, transactionID, msIndex)
+
+			// Subtract spent dust from diff
+			switch input.OutputType() {
+			case iotago.OutputSigLockedDustAllowanceOutput:
+				address := input.Address().String()
```

Comment:
you should convert the address bytes to string, but the String function uses hex encoding